TEMPLATE_README.md

```diff
@@ -0,0 +1,3 @@
+# {{name}}
+
+A project built with the Vapor web framework.
```

Comment:
Can we flesh this out a bit more with some commands to build and run the project?